crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/meta_net.rs

```diff
@@ -247,6 +247,9 @@ pub enum MetaNetCon {
         res: ResStore,
         tun: KitsuneP2pTuningParams,
     },
+
+    #[cfg(test)]
+    Test { notify_succeed: bool },
```

Comment:
I'm not sure about keeping this. I started prepping test tools for this task and it's sibling but this PR has got big enough and I want to get it in before starting on the next task because that's going to make this PR quite a bit bigger. This will be useful but it's going to need to be extended to other places. I can remove this and re-add but I'll leave it here unless I'm asked to do that :)